vcpkg/concepts/supported-hosts.md

```diff
@@ -54,66 +54,68 @@ The fully supported, tested platforms are:
 
 ### Windows
 
-* The latest Windows / Windows Server release. These are Windows 11 and Windows Server 2022 as of this writing.
-* The latest Visual Studio update, Visual Studio 2022 version 17.11 as of this writing.
+* The latest Windows / Windows Server release. These are Windows 11 and Windows Server 2025 as of this writing.
+* The latest Visual Studio update, Visual Studio 2022 version 17.14 as of this writing.
 
-### macOS
+### macOS (arm64 / "Apple Silicon")
 
 macOS is intended to track the latest version of macOS and contemporary version of Xcode Command Line tools. However,
-updating macOS machines is a manual process, and macOS frequently changes things in ways that break vcpkg's testing. As of this writing, we are using:
+updating macOS machines is a manual process, and macOS frequently changes things in ways that break vcpkg's testing.
+As of this writing, we are using:
 
-* macOS 14.5
-* XCode Command Line Tools 15.3
+* macOS 15.6.1
+* XCode Command Line Tools 16.4
 
 ### Linux
 
-* The latest LTS release of Ubuntu, currently 22.04.
+* The latest LTS release of Ubuntu, currently 24.04.
 
 ### Android
 
-* Linux 64-bit Android NDK version r26d
+* Linux 64-bit Android NDK version r28c
 
 ## Full support, expected
 
 ### Windows
 
-* Windows 8.1 / Windows Server 2016 and later
-* Visual Studio 2015 and later
+* Windows 10 / Windows Server 2019 and later
+* Visual Studio 2017 and later
 
 ### macOS
 
-* The latest version of macOS, minus 2 major versions. For example, the current version of macOS is macOS 15 Sequoia,
-so we expect vcpkg to work as far back as macOS 13 Ventura. This is intended to track with Apple's own support for
+* The latest version of macOS, minus 2 major versions. For example, the current version of macOS is macOS 26 Tahoe,
+so we expect vcpkg to work as far back as macOS 14 Sonoma. This is intended to track with Apple's own support for
 macOS.
 * Contemporary versions of the Xcode Command Line Tools for a given release of macOS.
+* macOS x64
 
 ### Linux
 
 We intend to support AMD64 builds of glibc-based Linuxes still in support from their distribution vendor released within
 the last 5 years. Examples:
 
-* Ubuntu 24.04, 22.04, and 20.04 are in support from Canonical and released within the last 5 years, so they
-are expected to work. Ubuntu 18.04 is still in support from Canonical but was released more than 5 years ago, so
-we no longer consider it fully supported. 20.10 is newer than 20.04, but we do not consider it fully supported because
+* Ubuntu 24.04, and 22.04 are in support from Canonical and released within the last 5 years, so they

```

Comment:
Note that these Linux version changes aren't actually a change in policy, I'm just updating the values since I touched this document anyway.